llvm/tools/llvm-exegesis/README.md

```diff
@@ -32,6 +32,8 @@ architectures:
     e.g. pseudo instructions and most register classes are not supported.
 * MIPS
 * PowerPC (PowerPC64LE only)
+* RISC-V
+  * Supported extensions: M, A, F, C, B, initial V.
```

Comment:
D?

Reply:
Reading this part again, I don't think we explicitly exclude any extensions, right? It's usually technical shortfalls that limit the kind of instructions we can measure / generate (e.g. we still can't measure memory instructions in RVV). So perhaps we can rephrase this part of the description.